network-protocols.md

```diff
@@ -375,3 +375,48 @@ type RetrievePieceChunk struct {
 ```
 
 TODO: document the query deal interaction
+
+# BlockSync
+
+The blocksync protocol is a small protocol that allows Filecoin nodes to request ranges of blocks from each other. It is a simple request/response protocol with a protocol ID of `/fil/sync/blk/0.0.0`. It uses CBOR-RPC.
+
+```go
+type BlockSyncRequest struct {
+    Start Cid
+    RequestLength uint64
```

Comment:
nit: can we call this `NumBlocks` or something that indicates we are talking about some number of blocks here?

Reply:
Maybe chain length would be better. NumBlocks would be more misleading due to EC not giving us a fixed number of blocks for a chain of given length